concourse.iml

```diff
@@ -10,14 +10,14 @@
     </content>
```

Comment:
Since this is a Maven project I don't think we should be checking in *.iml files.  IntelliJ can use the pom.xml file to manage dependencies.

Reply:
Removed 👍